pandas/_libs/tslibs/offsets.pyx

```diff
@@ -3350,7 +3350,13 @@ cdef class _CustomBusinessMonth(BusinessMixin):
         """
         Define default roll function to be called in apply method.
         """
-        cbday = CustomBusinessDay(n=self.n, normalize=False, **self.kwds)
+        if self.offset:
```

Comment:
isn't offset *always* defined?

Reply:
Yes, the purpose of the check here is to make sure we only pass in a zero `self.offset` into `CustomBusinessDay` (and that we do not do unnecessary copying of the kwd dict).